### PR TITLE
Big Red UnCASable Caves

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Big Red Caves cannot be hit by CAS anymore.

## Why It's Good For The Game

Every other cave in the game can't be hit by CAS, so Big Red's an outlier, this fixes that.

## Changelog

:cl: Geeves, Sose96
balance: Big Red caves can't be CAS'd anymore.
/:cl:

### CODE REVIEW

**Comment by Joelampost** [comment]
here before the cope :)

**Comment by Segrain** [comment]
Every other cave does not extend literally to LZ's walls.

**Comment by PhantomEpicness** [comment]
> Every other cave does not extend literally to LZ's walls.

Yeah the entrances should be fair game while the caves should be uncasable

**Comment by stanalbatross** [comment]
image

red highlighted areas near LZs should remain CASable

red highlighted cave entrances are er idk areas we will look at later

**Comment by hotimep** [comment]
no

**Comment by FoxyStalin** [comment]
Stan's comment was exactly what I was going to ask.
This is going to be a shame because CAS is fun and being able to use it for the entire round instead of 'Oh marines are in caves I guess I'll do nothing'  was nice, but, oh well

File: code/game/area/BigRed.dm [hunk 1]
```diff
@@ -634,3 +627,35 @@
...
+	icon_state = "garden"
+	ceiling = CEILING_UNDERGROUND_ALLOW_CAS
+
+/area/bigredv2/outside/filtration_cave_cas
+	name = "\improper filtration_cave_cas"
+	icon_state = "garden"
+	ceiling = CEILING_UNDERGROUND_ALLOW_CAS
+
+/area/bigredv2/outside/lambad_cave_cas
+	name = "\improper lambad_cave_cas"
```
Comment (stanalbatross) [inline code/game/area/BigRed.dm:659]: these need to be proper names as these show up constantly in game

also correct the misspelling of lambda

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/game/area/BigRed.dm (hunk 1 above)
Comment (Geevies) [reply code/game/area/BigRed.dm:659]: lambad

**Review by stanalbatross — APPROVED** [review]
code approved

**Comment by stanalbatross** [comment]
image
you need to also modify this nightmare insert

**Comment by FoxyStalin** [comment]
Static Coms being TM'd reminded me that that area around the north of Marshalls would need to have adjustments in the CASability as well,  defending the Comms Tower with no air support would suck.

**Review by stanalbatross — APPROVED** [review]

**Pull request merged**